app/subscription/singbox.py

```diff
@@ -256,6 +256,11 @@ def add(self, remark: str, address: str, inbound: dict, settings: dict):
         if net in ("kcp", "splithttp"):
             return
 
+        if net == "quic" and inbound["header_type"] != 'none':
+            return
+
+        if 
```

Comment:
what's this if? it's always better to have project in your code space and run a test before making PR. (as we don't have test actions sadly)

Reply:
Sorry, I have actually tested it, idk where this 'if' came from

Reply:
As i don't use Git, this problems occur